crates/ark/src/modules/rstudio/rstudioapi.R

```diff
@@ -1,20 +1,34 @@
 #' @export
 .rs.api.versionInfo <- function() {
-    # comments are what rstudioapi actually does
-    # https://github.com/rstudio/rstudio/blob/bb729e14867f6f95e26600d8b38e4551402cf7de/src/cpp/r/R/Api.R#L135-L145
-    # info <- list()
+
+    current_year <- format(Sys.Date(), "%Y")
+    positron_citation <-
```

Comment:
See here for what RStudio has for a citation: https://github.com/rstudio/rstudio/blob/main/src/cpp/session/resources/CITATION.in

Reply:
Minor, do you think RStudio should have used `https://posit.co/`?